lib/generate/generate.go

```diff
@@ -16,6 +19,22 @@ func WithSpecialChars(wordlist []string) []string {
 	return wordlist
 }
 
+func WithNumbers(wordlist []string, specificRange int) []string {
+	Numbers := make([]int, specificRange)
+	for _, word := range wordlist {
+		for range Numbers {
+			var randomPassword = word + strconv.Itoa(rand.IntN(specificRange))
```

Comment:
Why do you use a random number instead of going through all the numbers in the range ?

Reply:
Because after reviewing your issue, i saw that you want random number in the list and not a list with the range of this number. 

You can look at my first commit, i was thinking that way first, but the issue make me do it this way. 

**My first commit implemented it like** :
`--numbers -> 1,2,3,4,...,20`
**But after i saw your issue with this example output** : 
```
passfinder33
passfinder39
passfinder19
```

I change it in that way.

Reply:
Oh right, sorry if that wasn't clear. Yes, it's better if the list is complete and not random.

Reply:
No problem, i should have asked first 😅

So to sure, if the range is 20 the output will be like this ? 
(It doesn't start from 0)
```
passfinder1
passfinder2
[...]
passfinder20
```

Reply:
Ok ! All good, i tested it locally, update the feature and the test !